packages/api/src/SyndesisFetch.tsx

```diff
@@ -1,27 +1,28 @@
 import * as React from 'react';
 import { ApiContext } from './ApiContext';
-import { IRestRenderProps, Rest } from './Rest';
+import { IFetchRenderProps, Fetch } from './Fetch';
 import { Stream } from './Stream';
 
-export interface ISyndesisRestProps<T> {
+export interface ISyndesisFetchProps<T> {
```

Comment:
Does it make sense for this to extend `IFetchProps`?

Reply:
Since `SyndesisFetch` is about simplifying the usage of the underlying `Fetch` component I'd say no.